### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by morganizwd**

### CODE REVIEW

**Comment by morganizwd** [comment]
[Задания №1](https://trello.com/c/wR9ezMRF/64-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%961)

File: Task4/Task4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace ConsoleApp4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите трехзначное число: ");
+            int tripleNum = Convert.ToInt32(Console.ReadLine());
+            int firstNum = tripleNum / 100;
```
Comment (PaShkadun) [inline Task4/Task4/Program.cs:11]: Add whiteline between this lines.

File: Task4/Task4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            int secondNum = (tripleNum / 10) % 10;
+            int thirdNum = tripleNum % 10;
+            int max = firstNum;
+            if (secondNum > max)
```
Comment (PaShkadun) [inline Task4/Task4/Program.cs:15]: Add whiteline between this lines.

File: Task4/Task4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            {
+                max = secondNum;
+            }
+
+            if (thirdNum > max)
+            {
+                max = thirdNum;
+            }
+            Console.WriteLine($"Самая большая цифра в числе: {max} ");
```
Comment (PaShkadun) [inline Task4/Task4/Program.cs:24]: Add whiteline between this lines.

**Pull request merged**